Tests/AnyCodableTests/KeyedEncodingContainerProtocol+AnyCodableTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  KeyedEncodingContainerProtocol+AnyCodableTests.swift
+//  AnyCodableTests
+//
+//  Created by Dale Buckley on 22/03/2018.
+//
+
+import XCTest
+@testable import AnyCodable
+
+class KeyedEncodingContainerProtocolAnyCodableTests: XCTestCase {
+    
+    static let allTests = [
+        ("testEncodingEmptyAnyCodable", testEncodingEmptyAnyCodable),
+        ("testEncodingNonEmptyAnyCodable", testEncodingNonEmptyAnyCodable)
+    ]
+
+    func testEncodingEmptyAnyCodable() throws {
+
+        let mock = MockCodable(nil)
+        let data = try JSONEncoder().encode(mock)
+
+        XCTAssertFalse(String(data: data, encoding: .utf8)!.contains("\"any\":"))
```

Comment:
should that be checking `null` as well?

Reply:
The whole point is that `"any"` won't exist, so no. But I will add a test that takes an `NSNull` so we can test that too.

Reply:
I meant `"\"any\":null"`